Jenkinsfile

```diff
@@ -38,7 +36,7 @@ for (i = 0; i < buildTypes.size(); i++) {
         // First stage is actually checking out the source. Since we're using Multibranch
         // currently, we can use "checkout scm".
         stage('Checkout') {
-          checkout scm
+          infra.checkoutSCM()
```

Comment:
Using our [Pipeline shared library](https://github.com/jenkins-infra/pipeline-library/blob/0589e1b3eb64e7e5b9384b70b5d409e97a694912/vars/infra.groovy#L31-L44=) implementation enables long paths to avoid problems with tests on Windows agents.